src/EditorFeatures/Core.Wpf/StringIndentation/StringIndentationTag.cs

```diff
@@ -31,5 +37,30 @@ internal class StringIndentationTag : BrushTag
             var brush = view.VisualElement.TryFindResource("outlining.verticalrule.foreground") as SolidColorBrush;
             return brush?.Color;
         }
+
+        // Intentionally returning the base impl, we have never supported this facility, and there is no contract around
+        // placing these tags in sets or maps.
+        public override int GetHashCode()
+            => base.GetHashCode();
```

Comment:
> Or, move this one back to throwing if anybody tries to do this.

Gah :D  i moved away from throwing because i thought you didn't like it :D.  I much prefer throwing.